src/common/stringUtils.ts

```diff
@@ -0,0 +1,3 @@
+export const capitalize = (value: string) => {
```

Comment:
Do we need this? I think that this can be done with [css text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)